src/fluree/server/consensus/core.clj

```diff
@@ -66,36 +66,36 @@
    :state-atom     state-machine-atom
    :storage-read   (or storage-group-read
                        (io-file/connection-storage-read
-                         log-directory
-                         encryption-key))
+                        log-directory
```

Comment:
i think i prefer 2-space indentation for function calls when all the arguments are on a new line. i think it better expresses the semantic difference between the function and the arguments it takes. but i'm also ok with leaving it this way if other folks disagree.

Reply:
This is actually just cljfmt treating lists like other multiline compound data literals. For example:

```clojure
[:foo
 :bar
 :baz]
```

I'm not sure if you can tell it to make an exception for lists or if you'd have to make all such data literals indent w/ two spaces on subsequent lines.

Reply:
Looks like I was wrong and this is specifically configurable in cljfmt: https://github.com/weavejester/cljfmt#formatting-options (see `:function-arguments-indentation` section). I'm happy to go w/ whatever the team consensus is here.

Reply:
I like 2-space indentation here as well

Reply:
Do you all have a preference between the `:cursive` and `:zprint` options for that param?